tests/test_safe_tx_service_operator.py

```diff
@@ -80,11 +80,21 @@ def test_add_delegate(self, add_delegate_mock: MagicMock):
         delegate_address = Account.create().address
         label = "Test"
         signer = list(safe_operator.accounts)[0]
+
+        expected_hash = safe_operator.safe_tx_service.create_delegate_message_hash(
+            delegate_address
+        )
+        expected_signature_signed = signer.signHash(expected_hash)
```

Comment:
```suggestion
        expected_signature = signer.signHash(expected_hash)
```
Looks like a bit redundant signature is supposed to be signed :)

Reply:
Yes, you are right. Updated https://github.com/safe-global/safe-cli/pull/427/commits/b8f86ad562d947a4f540256bfde2e90f578331ac